single-audit/resolve_findings/models.py

```diff
@@ -53,7 +53,17 @@ def __str__(self):
         return self.name
 
 
+class NewFindingManager(models.Manager):
+    def get_queryset(self):
+        return super(NewFindingManager,
+                     self).get_queryset()\
+                     .filter(status='new')
+
+
 class Finding(models.Model):
+    objects = models.Manager()
+    new = NewFindingManager()
```

Comment:
Learning about the Manager pattern in Django ... this is neat! Basically seems like a way to encapsulate DB-query-related logic for each model in a predictable place?